test/jasmine/tests/axes_test.js

```diff
@@ -6901,6 +6902,18 @@ describe('Test axes', function() {
             expect(positions).toEqual(expPositions);
         }
 
+        function _assertClose(expPositions) {
+            var ax = gd._fullLayout.xaxis;
+
+            // minor positions
+            var positions =
+                ax._vals
+                    .filter(function(d) { return d.minor; })
+                    .map(function(d) { return d.x; });
+
+            expect(positions).toBeCloseToArray(expPositions);
```

Comment:
Can you give this call a precision, so it's explicit how close we're testing? This second arg to `toBeCloseToArray` is essentially digits to match after the decimal point, so a good number given your usage would be something like 3, unless there's a good reason to use a worse precision.

Reply:
done!